source/time.tex

```diff
@@ -3662,13 +3634,13 @@
 
 \begin{itemdescr}
 \pnum
-\remarks
-This function does not participate in overload resolution unless
+\constraints
 \tcode{SourceClock::to_sys(t)} is well-formed.
```

Comment:
I think I remember one such change from `\remarks` to `\constraints` also changed "well-formed" to "valid".

Reply:
"well-formed" is a defined term; "valid" is not. We should at least pick one and apply it consistently. Filed #3499 to track that.